coretex/cli/config.py

```diff
@@ -117,19 +134,23 @@ def configNode() -> None:
     click.echo("Storage path should be the same as (if) used during --user config")
     storagePath = click.prompt("Storage path (press enter to use default)", Path.home() / ".coretex", type = str)
 
-    image = arrowPrompt(["gpu", "cpu"])
+    gpuExists = isGPUAvailable()
+    if gpuExists:
```

Comment:
`if isGPUAvailable()`, no need to create a new variable since it is not used anywhere else.